lib/index.js

```diff
@@ -82,7 +82,7 @@ function SpmAgent (spmUrl) {
   }.bind(this))
   this.influxSender.on('error', function (e) {
     counters.error += 1
-    logger.error('Error in spm-sender ' + JSON.stringify(e))
+    logger.error('Error in influx-sender ' + JSON.stringify(e))
```

Comment:
Here we are replacing spm-sender with influx-sender, but above I see the opposite?

Reply:
Yes. It looks the influx error handler was installed twice and had the wrong log message. It was probably a copy/paste issue.